src/e2e/app/cli/repo-tests/src/commands/test_sql_command.rs

```diff
@@ -42,19 +35,6 @@ async fn test_datafusion_cli() {
         ),
         "Unexpected output:\n{stdout}",
     );
-
-    assert.success();
-}
-
-#[test_group::group(engine, datafusion)]
-#[test_log::test(tokio::test)]
-async fn test_datafusion_cli_not_launched_in_root_ws() {
```

Comment:
Reason for deletion: if files appear, we will catch them by `Check generated files` CI step

Reply:
This test is meant to catch situation where a developer accidentally created `.kamu` workspace in the root of the `kamu-cli` folder. This leads to all kinds of misleading errors and Olena spent a whole day once debugging this issue. So it would be nice to keep this test.

We don't have this problem in CI at all - this is important for local development only.

Reply:
Done